src/backend/sgx/ioctls.rs

```diff
@@ -192,6 +192,7 @@ mod tests {
     use super::*;
 
     #[test]
+    #[ignore]
```

Comment:
Why are these being ignored rather than using the cfg approach?

Reply:
I removed these changes from this patch. `#[tests]` is prepended with `host_can_test_sgx`.